app/assets/stylesheets/application.css.scss

```diff
@@ -192,3 +203,6 @@ section {
     }
   }
 }
+
+@import 'patients';
```

Comment:
The rails asset pipeline automatically loads these files, so we don't need to double-include them here. (See section 2.1 at http://guides.rubyonrails.org/asset_pipeline.html, specifically the stuff about `require_tree`.) 

Can you remove these two imports?